src/common/actions/repository-input.js

```diff
@@ -0,0 +1,69 @@
+import 'isomorphic-fetch';
+import parseGitHubUrl from 'parse-github-url';
+
+export const CHANGE_REPOSITORY_INPUT = 'CHANGE_REPOSITORY_INPUT';
```

Comment:
`CHANGE_REPOSITORY_INPUT` sounds alright to me, given that it's going to be called every time the value is changed.

Suggested alternatives: `REPOSITORY_INPUT_ON_CHANGE`, `UPDATE_REPOSITORY_INPUT`